src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,37 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
+
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        DateTimeFormatter formatter = DateTimeFormatter.ofPattern("dd.MM.yyyy");
+        final String resultSeparator = " - ";
+        final int indDate = 0;
+        final int indName = 1;
+        final int indHours = 2;
+        final int indMoney = 3;
```

Comment:
Please move variables that never changed to class constants with corresponding naming DATE_INDEX, NAME_INDEX...